sdk/servicefabric/servicefabric/package.json

```diff
@@ -2,10 +2,10 @@
   "name": "@azure/servicefabric",
   "author": "Microsoft Corporation",
   "description": "ServiceFabricClient Library with typescript type definitions for node.js and browser.",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
Did we change this manually or pass in the version to the autorest command?
The command should have updated the version in the context file: https://github.com/Azure/azure-sdk-for-js/blob/ec88a52f808aa0080efd4f07bd196a6adbefd1f0/sdk/servicefabric/servicefabric/src/serviceFabricClientContext.ts#L15

Reply:
Good catch I did this manually. Re-generating with --package-version parameter instead